anda/misc/prime-run/prime-run.spec

```diff
@@ -0,0 +1,38 @@
+Name:           prime-run
+Version:        0.0.1
+Release:        1%{?dist}
+Summary:        A simple script to run an application with NVIDIA PRIME GPU offloading
+
+License:        MIT
+URL:            https://terra.fyralabs.com
+Source0:        prime-run.sh
+
+# Bash script
+Requires:       bash
+BuildArch:      noarch
+Supplements:    nvidia-driver
```

Comment:
`Supplements:` only work if `nvidia-driver` requires this package; in this case there's probably no effect